_includes/v21.1/import-into-offline.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_info}}
```

Comment:
tbh I feel like this creeps closer to a warning to me than just "info".  meta: wish we had the middle ground "yellow caution note" for these situations